### DISCUSSION TITLE
Building IronOS from source for Pinecil, using the Dockerfile: Fatal Error

### DISCUSSION THREAD

**Discussion by squaregrooper** [Ralim/IronOS#1268] [Q&A] [answered]
After running `make model=Pinecil firmware-EN`, I had an error about an 'include' directory not being in the `~/IronOS/source/Core/Drivers/usb-pd/include` directory.  After adding this folder, im getting the

`Core/Drivers/USBPD.cpp:7:10: fatal error: fusb302b.h: No such file or directory
    7 | #include "fusb302b.h"` error

Can someone help me out with this? I made it this far, and figured i was missing a parameter in my 'make' command, but wasnt sure.

**Comment by Ralim** [comment, marked as answer]
You did not `git clone --recursive` so you do not have the sub-repo.
You will need to remove the folder you made and run a `git submodule init && git submodule update` to fetch the sub-repo

**Comment by squaregrooper** [comment]
thank you!
